meta/system.py

```diff
@@ -287,6 +287,41 @@ def gt_features(self, corpus, prefix="", **kwargs):
         self.feature_flows[corpus]["gt"] = features.basic_cache_flow(feature_path)
         self.feature_flows[corpus]["uncached_gt"] = f.feature_flow
 
+    def generic_features(
+        self, corpus, name, feature_flow, port_name="features", prefix="", **kwargs
+    ):
+        """
+        :param str corpus: corpus identifier
+        :param str name: feature identifier, like "mfcc". Also used in the naming of the output feature caches.
+        :param rasr.FlowNetwork feature_flow: definition of the RASR feature flow network
+        :param str port_name:
+                Maps the output of a flow network (key) to the name of the extracted feature (value).
+                name is the value and port_name is the key.
+                The port_name_mapping is used as parameter for the
+                FeaturesExtractionJob which normally has this structure:
+                {"features": name}
```

Comment:
```suggestion
        :param str port_name: output port of the flow network to use
```

Reply:
Now that the user does not specify the port_name_mapping herself it's fine to not explain it anyomore.